src/diffusers/models/transformers/__init__.py

```diff
@@ -5,6 +5,7 @@
     from .dit_transformer_2d import DiTTransformer2DModel
     from .dual_transformer_2d import DualTransformer2DModel
     from .hunyuan_transformer_2d import HunyuanDiT2DModel
+    from .latte_transformer_3d import LatteTransformer3DModel
```

Comment:
Feel free to add a little doc page for it. Example: https://github.com/huggingface/diffusers/blob/main/docs/source/en/api/models/hunyuan_transformer2d.md.

Reply:
Thanks. I have added a doc page for Latte.